docs/ops/logical/LogicalAnd_1.md

```diff
@@ -29,21 +29,20 @@ o_{i} = a_{i} \wedge b_{i}
 
 **Inputs**
 
-* **1**: A tensor of type *T* and arbitrary shape. **Required.**
-* **2**: A tensor of type *T* and arbitrary shape. **Required.**
+* **1**: A tensor of type *T_BOOL* and arbitrary shape. **Required.**
+* **2**: A tensor of type *T_BOOL* and arbitrary shape. **Required.**
 
 **Outputs**
 
-* **1**: The result of element-wise *LogicalAnd* operation. A tensor of type boolean.
```

Comment:
I believe the previous *LogicalAnd* here was correct (not "logical AND")

Reply:
You're right - it was an oversight during merging with master. Fixed.